frontend/src/components/common/authorized-wrapper/styles.module.scss

```
@@ -1,4 +1,9 @@
 .content {
   display: flex;
   flex-grow: 1;
+  min-height: 0;
+}
+
+.main_content {
+  overflow: scroll;
```

`overflow: scroll` will always show scroll bars (both vertical and horizontal),, but we want to show only vertical scroll and only in case of container's contents take more place then the container's height
```suggestion
.mainContent {
  overflow-y: auto;
```